web/src/scheduled_messages.js

```diff
@@ -16,6 +17,8 @@ import * as stream_data from "./stream_data";
 import * as timerender from "./timerender";
 
 export const MINIMUM_SCHEDULED_MESSAGE_DELAY_SECONDS = 5 * 60;
+export const SCHEDULING_MODAL_UPDATE_INTERVAL_IN_MILLISECONDS = 1000 * 60;
```

Comment:
Since it's 60 seconds * 1000 milliseconds, I'd flip these two numbers (like the line above, with 5 minutes * 60 seconds).